code/src/MaksLessonApp.js

```diff
@@ -0,0 +1,98 @@
+import React, { Component, useState} from "react";
+import Form from "components/Form";
+
+export const App = () => {
+    return <Form />;
+};
+
+
+
+
+
+// export const App = () => {
```

Comment:
I believe you can hide this file from showing on Github with the Git ignore (.gitignore) file if you want.